src/utils/constants.js

```diff
@@ -4,6 +4,8 @@ export const WETH_ADDRESS = "0xd0A1E359811322d97991E03f863a0C30C2cF029C";
 export const USDC_ADDRESS = "0xe22da380ee6B445bb8273C81944ADEB6E8450422";
 export const COIN_BASE_ASSET_PRICE_URL="https://api.coinbase.com/v2/prices/ETH-USD/spot"
 
+export const APY_API_PATH = "/token_apy?token_ticker=USDC&reward_token_ticker=ETH"
```

Comment:
The API path should only be `/token_apy`
token_ticker and reward_token_ticker should be appended to this variable wherever its used as params

Reply:
Think about it this way -- what if I say we need the APY for BTC or any other token (other than ETH) ?
Would you simply keep creating all these different variables or
 create 1 variable that has the api endoint and pass the required params to it (reusable)?